tests/functional/syntax/test_addmulmod.py

```diff
@@ -25,3 +26,24 @@ def foo() -> uint256:
 @pytest.mark.parametrize("code,exc", fail_list)
 def test_add_mod_fail(assert_compile_failed, get_contract, code, exc):
     assert_compile_failed(lambda: get_contract(code), exc)
+
+
+valid_list = [
+    """
+FOO: constant(uint256) = 3
+BAR: constant(uint256) = 5
+BAZ: constant(uint256) = 19
+BAX: constant(uint256) = uint256_addmod(FOO, BAR, BAZ)
```

Comment:
i think typically in computer science lore the next name in the list is `QUX` :D

Reply:
but i like BAX too tbh

Reply:
> i think typically in computer science lore the next name in the list is QUX :D

Oops I was not aware of this!